src/app/shared/components/cluster-summary/template.html

```diff
@@ -14,1088 +14,1107 @@
-          <ng-container *ngIf="vmwareclouddirector.csi">
-            <km-property *ngIf="vmwareclouddirector.csi.storageProfile">
-              <div key>Storage Profile</div>
-              <div value>{{vmwareclouddirector.csi.storageProfile}}</div>
-            </km-property>
-            <km-property *ngIf="vmwareclouddirector.csi.filesystem">
-              <div key>Filesystem</div>
-              <div value>{{vmwareclouddirector.csi.filesystem}}</div>
-            </km-property>
-          </ng-container>
-        </ng-container>
+              <!-- GCP Labels -->
+              <km-property *ngIf="displayTags(machineDeployment.spec.template.cloud?.gcp?.labels)">
+                <div key>Provider Labels</div>
+                <div value>
+                  <km-labels [labels]="machineDeployment.spec.template.cloud.gcp.labels"></km-labels>
+                </div>
+              </km-property>
 
-      </ng-container>
+              <!-- VSphere Tags -->
+              <km-property *ngIf="machineDeployment.spec.template.cloud?.vsphere?.tags?.length>0">
+                <div key>Provider Tags</div>
+                <div value>
+                  <km-labels [labels]="convertVSphereTagsIntoObject(machineDeployment.spec.template.cloud?.vsphere?.tags)"></km-labels>
+                </div>
+              </km-property>
 
-      <!-- KubeVirt -->
-      <ng-container *ngIf="cluster.spec.cloud?.kubevirt?.preAllocatedDataVolumes as dataVolumes">
-        <div fxLayout="row">
-          <div fxFlex="90%">
-            <km-kubevirt-pre-allocated-data-volumes [dataVolumes]="dataVolumes"></km-kubevirt-pre-allocated-data-volumes>
+            </div>
           </div>
         </div>
       </ng-container>
     </div>
-  </div>
 
-  <div [fxFlex]="flipLayout ? '100%' : '50%'"
-       fxLayout="column"
-       class="step">
-    <div class="header counter-5">Applications</div>
-    <div class="applications-summary"
-         [fxLayoutAlign]="flipLayout ? ' center' : ''">
-      <km-application-list *ngIf="applications?.length; else noApplications"
-                           [applications]="applications"
-                           [canEdit]="false"
-                           [cluster]="cluster"
-                           [view]="ApplicationsListView.Summary"></km-application-list>
-      <ng-template #noApplications>
-        No application selected.
-      </ng-template>
+    <div class="step">
+      <div class="header counter-5">Applications</div>
+      <div class="applications-summary"
+           [fxLayoutAlign]="flipLayout ? ' center' : ''">
+        <km-application-list *ngIf="applications?.length; else noApplications"
+                             [applications]="applications"
+                             [canEdit]="false"
+                             [cluster]="cluster"
+                             [view]="ApplicationsListView.Summary"></km-application-list>
+        <ng-template #noApplications>
+          <div class="no-applications">
```

Comment:
There are no styles attached to `no-applications` so why do we need this extra `div`?